conversions/rec_to_pol.py

```diff
@@ -0,0 +1,33 @@
+import doctest
+import math
+
+
+def rec_to_pol(real: float, img: float) -> tuple:
```

Comment:
```suggestion
def rectangular_to_polar(real: float, img: float) -> tuple[float, float]:
```
or `cartesian_to_polar`